src/storage/core.rs

```diff
@@ -235,11 +235,14 @@ where
     /// NOTICE! First write into storage without active blob may take more time due to active blob
     /// creation
     /// # Examples
-    /// ```no-run
+    /// ```no_run
+    /// use pearl::{Builder, Storage, UnitKey};
+    ///
     /// async fn write_data() {
-    ///     let key = 42u64.to_be_bytes().to_vec();
+    ///     let key = UnitKey::default();
     ///     let data = b"async written to blob".to_vec();
-    ///     storage.write(key, data).await
+    ///     let storage: Storage<UnitKey> = Builder::new().build().unwrap();
```

Comment:
I think `storage` could be an argument of the function. In this way it will be a little bit more practical example